### ISSUE TITLE
Confusion about SmoothL1LossOHEM layer when train without ohem

### ISSUE THREAD

**Issue by hzh8311** [YuwenXiong/py-R-FCN#73]
when I train the ResNet-50 without OHEM, and at the very end of the network outputs `loss_bbox` using `SmoothL1LossOHEM` layer with `pre_fixed_normalizer: 128`. It confuse me and I want to know what function does the `normalizer` have (in both w/ ohem and w/o ohem training)? Thank you for your attention.

**Comment by liyi14** [comment]
The sum of loss of each instance will be divided by a factor `normalization_` [like this](https://github.com/Microsoft/caffe/blob/1a2be8ecf9ba318d516d79187845e90ac6e73197/src/caffe/layers/softmax_loss_ohem_layer.cu#L59), typically use the batch_sizeas as this normalize factor. Since when training with OHEM, the batch_size(300) is different from the number of samples (128) that really generate loss, we use pre_fixed_normalizer to deal with conflicting.
